production/README.md

```diff
@@ -3,28 +3,13 @@
 There are a few ways to run the Grafana Agent, in order from
 easiest to hardest:
 
-- [Use the Install Script for Kubernetes](#install-script-for-kubernetes)
 - [Windows Installation](#windows-installation)
 - [Run the Agent with Docker](#running-the-agent-with-docker)
 - [Run the Agent locally](#running-the-agent-locally)
 - [Use the example Kubernetes configs](#use-the-example-kubernetes-configs)
 - [Build the Agent from Source](#build-the-agent-from-source)
 - [Use our production Tanka configs](#use-our-production-tanka-configs)
```

Comment:
Can we also find a place to link to the Grafana docs here?

Reply:
Good call, done!